src/access/tests/TestAccessRestrictionProvider.cpp

```diff
@@ -470,5 +529,85 @@ TEST_F(TestAccessRestriction, CombinedRestrictionTest)
     RunChecks(combinedRestrictionTestData, ArraySize(combinedRestrictionTestData));
 }
 
+TEST_F(TestAccessRestriction, AttributeStorageSeperationTest)
+{
+    std::vector<AccessRestrictionProvider::Entry> commissioningEntries;
+    AccessRestrictionProvider::Entry entry1;
+    entry1.fabricIndex    = 1;
+    entry1.endpointNumber = 1;
+    entry1.clusterId      = kWiFiNetworkManagementCluster;
+    entry1.restrictions.push_back({ .restrictionType = AccessRestrictionProvider::Type::kAttributeWriteForbidden });
+    entry1.restrictions[0].id.SetValue(1);
+    commissioningEntries.push_back(entry1);
+    EXPECT_EQ(accessRestrictionProvider.SetCommissioningEntries(commissioningEntries), CHIP_NO_ERROR);
+
+    std::vector<AccessRestrictionProvider::Entry> entries;
+    AccessRestrictionProvider::Entry entry2;
+    entry2.fabricIndex    = 2;
+    entry2.endpointNumber = 2;
+    entry2.clusterId      = kThreadBorderRouterMgmtCluster;
+    entry2.restrictions.push_back({ .restrictionType = AccessRestrictionProvider::Type::kCommandForbidden });
+    entry2.restrictions[0].id.SetValue(2);
+    entries.push_back(entry2);
+    EXPECT_EQ(accessRestrictionProvider.SetEntries(2, entries), CHIP_NO_ERROR);
+
+    auto commissioningEntriesFetched = accessRestrictionProvider.GetCommissioningEntries();
+    std::vector<AccessRestrictionProvider::Entry> arlEntriesFetched;
+    EXPECT_EQ(accessRestrictionProvider.GetEntries(2, arlEntriesFetched), CHIP_NO_ERROR);
+
+    EXPECT_NE(commissioningEntriesFetched[0].fabricIndex, arlEntriesFetched[0].fabricIndex);
+    EXPECT_NE(commissioningEntriesFetched[0].endpointNumber, arlEntriesFetched[0].endpointNumber);
+    EXPECT_NE(commissioningEntriesFetched[0].clusterId, arlEntriesFetched[0].clusterId);
+    EXPECT_NE(commissioningEntriesFetched[0].restrictions[0].restrictionType, arlEntriesFetched[0].restrictions[0].restrictionType);
+    EXPECT_NE(commissioningEntriesFetched[0].restrictions[0].id, arlEntriesFetched[0].restrictions[0].id);
+}
+
+constexpr CheckData listSelectionDuringCommissioningData[] = {
```

Comment:
- `kListXxx` --> `k` prefix for constants

Reply:
Please let me know if you feel strongly about this.  It doesnt match others like AccessControlTest.

Reply:
Added to https://github.com/project-chip/connectedhomeip/issues/35165